packages/hub/prisma/schema.prisma

```diff
@@ -325,3 +332,19 @@ model RelativeValuesPairCache {
 
   @@index([exportId])
 }
+
+model Searchable {
+  id String @id @default(cuid())
+
+  model   Model?  @relation(fields: [modelId], references: [id], onDelete: Cascade)
+  modelId String? @unique
+
+  definition   RelativeValuesDefinition? @relation(fields: [definitionId], references: [id], onDelete: Cascade)
```

Comment:
I don't think that Definitions are very key - I want to refactor them - but it's fine like this.